.vsconfig

```diff
@@ -10,6 +10,7 @@
     "Microsoft.VisualStudio.Component.Windows10SDK.20348",
     "Microsoft.VisualStudio.ComponentGroup.UWP.VC",
     "Microsoft.VisualStudio.Component.VC.Runtimes.x86.x64.Spectre", 
```

Comment:
```suggestion
    "Microsoft.VisualStudio.ComponentGroup.UWP.VC",
    "Microsoft.VisualStudio.Component.UWP.VC.ARM64", 
    "Microsoft.VisualStudio.Component.VC.Runtimes.ARM64.Spectre",
    "Microsoft.VisualStudio.Component.VC.Runtimes.x86.x64.Spectre", 
```

This is a good opportunity to add the ARM64-specific workloads that are missing.

Reply:
Great callout!